doc/tfsformat.rst

```diff
@@ -0,0 +1,142 @@
+The Table File System Format
+============================
+
+**TFS** is an acronym for "Table File System", and the TFS format has been used at `CERN <https://home.cern/>`_ since the LEP control system.
+The accelerator physics codes `MAD-X <https://madx.web.cern.ch/>`_ and `MAD-NG <https://madx.web.cern.ch/releases/madng/html/>`_ output tables in the TFS format, and accept TFS files as input for tables.
+
+The origin of the TFS definition [#f1]_ [#f2]_ seems more than difficult to find, and no official spec has been maintained since then.
+As such, this document aims to provide a comprehensive description of the TFS format, at the very least as it is parsed and written by `tfs-pandas`.
+
+Written TFS files follow a structure in two parts: first the ``headers`` lines and then the ``table`` itself.
+They are described below, including their specifics.
+
+Headers
+-------
+
+The first section of the TFS file contains `headers`, lines each containing a parameter, its type and its value.
+All headers lines start with the `@` character, and have the following structure:
+
+- The `@` character, to identify the line as a header.
+- The name of the parameter.
+- The type identifier of the parameter (see `Types and Identifiers`_ below).
+- The value of the parameter.
+
+An example header line is:
+
+.. code-block::
+
+    @ NAME                   %le             0.269975
```

Comment:
Very pedantic, but would this ever be an example? Surely `NAME` would be a string?

Reply:
Absolutely, good catch.